great_expectations/render/__init__.py

```diff
@@ -1 +1,275 @@
+
+
+class LegacyPrescriptiveRendererType(str, Enum):
+    """Available legacy prescriptive renderer names"""
+
+    SUMMARY = ".".join([LegacyRendererType.PRESCRIPTIVE, "summary"])
+
+
+class LegacyDiagnosticRendererType(str, Enum):
+    """Available legacy diagnostic renderer names"""
+
+    META_PROPERTIES = ".".join([LegacyRendererType.DIAGNOSTIC, "meta_properties"])
+    OBSERVED_VALUE = ".".join([LegacyRendererType.DIAGNOSTIC, "observed_value"])
+    STATUS_ICON = ".".join([LegacyRendererType.DIAGNOSTIC, "status_icon"])
+    SUMMARY = ".".join([LegacyRendererType.DIAGNOSTIC, "summary"])
+    UNEXPECTED_STATEMENT = ".".join(
+        [LegacyRendererType.DIAGNOSTIC, "unexpected_statement"]
+    )
+    UNEXPECTED_TABLE = ".".join([LegacyRendererType.DIAGNOSTIC, "unexpected_table"])
+
+
+class LegacyDescriptiveRendererType(str, Enum):
+    """Available legacy descriptive renderer names"""
+
+    COLUMN_PROPERTIES_TABLE_DISTINCT_COUNT_ROW = ".".join(
+        [
+            LegacyRendererType.DESCRIPTIVE,
+            "column_properties_table",
+            "distinct_count_row",
+        ]
+    )
+    COLUMN_PROPERTIES_TABLE_DISTINCT_PERCENT_ROW = ".".join(
+        [
+            LegacyRendererType.DESCRIPTIVE,
+            "column_properties_table",
+            "distinct_percent_row",
+        ]
+    )
+    COLUMN_PROPERTIES_TABLE_MISSING_COUNT_ROW = ".".join(
+        [LegacyRendererType.DESCRIPTIVE, "column_properties_table", "missing_count_row"]
+    )
+    COLUMN_PROPERTIES_TABLE_MISSING_PERCENT_ROW = ".".join(
+        [
+            LegacyRendererType.DESCRIPTIVE,
+            "column_properties_table",
+            "missing_percent_row",
+        ]
+    )
+    COLUMN_PROPERTIES_TABLE_REGEX_COUNT_ROW = ".".join(
+        [LegacyRendererType.DESCRIPTIVE, "column_properties_table", "regex_count_row"]
+    )
+    EXAMPLE_VALUES_BLOCK = ".".join(
+        [LegacyRendererType.DESCRIPTIVE, "example_values_block"]
+    )
+    HISTOGRAM = ".".join([LegacyRendererType.DESCRIPTIVE, "histogram"])
+    QUANTILE_TABLE = ".".join([LegacyRendererType.DESCRIPTIVE, "quantile_table"])
+    STATS_TABLE_MAX_ROW = ".".join(
+        [LegacyRendererType.DESCRIPTIVE, "stats_table", "max_row"]
+    )
+    STATS_TABLE_MEAN_ROW = ".".join(
+        [LegacyRendererType.DESCRIPTIVE, "stats_table", "mean_row"]
+    )
+    STATS_TABLE_MIN_ROW = ".".join(
+        [LegacyRendererType.DESCRIPTIVE, "stats_table", "min_row"]
+    )
+    VALUE_COUNTS_BAR_CHART = ".".join(
+        [LegacyRendererType.DESCRIPTIVE, "value_counts_bar_chart"]
+    )
+
+
+class RenderedAtomicValue(DictDot):
+    def __init__(
+        self,
+        schema: Optional[dict] = None,
+        header: Optional["RenderedAtomicValue"] = None,
+        template: Optional[str] = None,
+        params: Optional[dict] = None,
+        header_row: Optional[List["RenderedAtomicValue"]] = None,
+        table: Optional[List[List["RenderedAtomicValue"]]] = None,
```

Comment:
```suggestion
        table: Optional[List[List[RenderedAtomicValue]]] = None,
```
Do you still need to use the string version of these if we are using `from __future__ import annotations`?

Reply:
✅